crates/kubelet/src/node.rs

```diff
@@ -4,8 +4,34 @@ use k8s_openapi::api::coordination::v1::Lease;
 use k8s_openapi::api::core::v1::Node;
 use k8s_openapi::apimachinery::pkg::apis::meta::v1::Time;
 use kube::api::{Api, DeleteParams, PatchParams, PostParams};
+use kube::Error;
 use log::{debug, error, info};
 
+macro_rules! retry {
+    ($action:expr, times: $num_times:expr, $on_err:expr) => {{
+        let mut n = 0u8;
```

Comment:
For my own edification, why is this specified this way instead of just `1`?

Reply:
Because we increment `n` at the beginning of the loop instead of the end. I can refactor it to start at 1 if you want.

Reply:
It doesn't matter too much to me, I was just curious on the reasoning